### DISCUSSION TITLE
Setting option values

### DISCUSSION THREAD

**Discussion by xq114** [xmake-io/xmake#2604] [Q&A] [answered]
https://github.com/xmake-io/xmake/blob/master/xmake/core/project/option.lua

目前我只找到有一个`option:enable()`接口可以调整值，但这个接口描述是改成true或者false的。要更改一般字符串option的值，如何实现？

**Comment by waruqi** [comment, marked as answer]
`opt:set_value("xxx")` and `opt:value()`
